install_config/uninstall-olm-using-ansible.adoc

```diff
@@ -0,0 +1,15 @@
+// This assembly is included in the following assemblies:
+//
+// * n/a
+
+:_content-type: ASSEMBLY
```

Comment:
Thanks for splitting this file! You need to add it to the `_topic_map.yml` file so it'll build.

Reply:
Addressed